File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L1/07/Kasus6L1.py
import sys

# Deklarasi
num = [0] * 10

# for untuk 10 kali input
for i in range(10):
    # Read a number
    print("Read a number: ", end="")
    num[i] = int(input())

# Display the array
for i in range(9, -1, -1):
    # Print angka dalam array
    print(num[i])

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-06/plagiarized/L1/07/Kasus6L1.kt
package IR

/**
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
fun main(args: Array<String>) {
    // Deklarasi
    val input = java.util.Scanner(System.`in`)
    val num = IntArray(10)
    // for untuk 10 kali input
    for (i in 0 until 10) {
        // Read a number
        print("Read a number: ")
        num[i] = input.nextInt()
    }
    // Display the array
    for (i in 9 downTo 0) {
        // Print angka dalam array
        println(num[i])
    }
}

